components/CardThumb.js

```diff
@@ -0,0 +1,43 @@
+/*Outer dependencies*/
+import React from 'react';
+import { Pressable } from 'react-native';
+import styled from 'styled-components/native';
+
+/*Styled components*/
+const ThumbContainer = styled.View`
+  background: green;
+`;
+
+const ThumbText = styled.Text`
+  color: blue;
+`;
+
+const ThumbImage = styled.Image`
+  width: 100px;
+  height: 100px;
+`;
+
+export const CardThumb = (props) => {
+  /*Deconstructing props*/
+  const { card, navigation } = props;
+
+  /*Function to create route for CardDetails when clicking the entire CardThumb*/
+  const onPressCardButton = () => {
+    navigation.navigate('CardDetails', { card: card });
+  };
+
+  return (
+    <>
```

Comment:
you are returning a single component here, so there is no need to using react fragment.

Reply:
Done!